0-bootstrap/provider.tf

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Uncomment this provider definition in order to be able to perform the Groups Creation
+ *provider "google-beta" {
```

Comment:
Any issue having this uncommented by default?

Reply:
The module terraform-google-modules/group/google uses the resource google_cloud_identity_group, and this resource requires that we setup a billing_project on the provider level. 
As we can't have a default value for a billing_project, when the user sets the groups.create_groups with true and no billing_project, it will throw the error:  
"Error creating Group: googleapi: Error 403: Your application has authenticated using end user credentials from the Google Cloud SDK or Google Cloud Shell which are not supported by the cloudidentity.googleapis.com. We recommend configuring the billing/quota_project setting in gcloud or using a service account through the auth/impersonate_service_account setting.".

To follow your suggestion to remove this uncommented by default what I did was: I added a new validation that checks if the user is passing the billing_project when the groups.create_groups is true.